apps/freelance/src/pages/Freelancer/CreateProfile1/index.tsx

```diff
@@ -0,0 +1,135 @@
+import workExperience from "utils/select-options/work-experience.json";
+import employmentType from "utils/select-options/employment-type.json";
+import englishLevel from "utils/select-options/english-level.json";
+
+export function CreateProfile1() {
+	const { t } = useTranslation();
+
+	return (
+		<ThemeProvider theme={ThemeColors && ThemeBackground}>
+			<StyledPage>
+				<Dashboard userRole="freelancer">
+					<StyledTitle tag="h2" fontSize="md" fontWeight={700}>
+						{t("dashboard.profilePage.title")}
+					</StyledTitle>
+					<Form>
+						<input
+							id="fullName"
+							type="text"
+							name="fullName"
+							placeholder={t("freelancer.createProfile.fullNamePlaceholder")}
+							required
+						/>
+						<div className="selectContainer">
+							<select id="country" name="country" required>
+								<option value="" disabled selected hidden>
+									{t("freelancer.createProfile.selectOption.country")}
+								</option>
+								{countries.map(({ name }) => {
+									return <option>{name}</option>;
+								})}
+							</select>
+						</div>
+						<div className="selectContainer">
+							<select id="category" name="category" required>
+								<option value="" disabled selected hidden>
+									{t("freelancer.createProfile.selectOption.category")}
+								</option>
+								{categories.map(({ name }) => {
+									return <option>{name}</option>;
+								})}
+							</select>
+						</div>
+						<div className="selectContainer">
+							<select id="hourRate" name="hourRate" required>
+								<option value="" disabled selected hidden>
+									{t("freelancer.createProfile.selectOption.hourRate")}
+								</option>
+								{hourRate.map(({ name }) => {
+									return <option>{name}</option>;
+								})}
+							</select>
+						</div>
+						<input
+							id="position"
+							type="text"
+							name="position"
+							placeholder={t("freelancer.createProfile.positionPlaceholder")}
+							required
+						/>
+						<div className="selectContainer">
+							<select id="amountHours" name="amountHours" required>
+								<option value="" disabled selected hidden>
+									{t("freelancer.createProfile.selectOption.amountHours")}
+								</option>
+								{workHours.map(({ name }) => {
+									return <option>{name}</option>;
+								})}
+							</select>
+						</div>
+						<div className="selectContainer">
+							<select id="skills" name="skills" required>
+								<option value="" disabled selected hidden>
+									{t("freelancer.createProfile.selectOption.skills")}
+								</option>
+								{skills.map(({ name }) => {
+									return <option>{name}</option>;
+								})}
+							</select>
+						</div>
+						<div className="selectContainer">
+							<select id="workExperience" name="workExperience" required>
+								<option value="" disabled selected hidden>
+									{t("freelancer.createProfile.selectOption.workExperience")}
+								</option>
+								{workExperience.map(({ name }) => {
```

Comment:
will you always have workExperience array?

Reply:
Yes, as all the options, there will be a list to select one option from the list. Same as the first review, I am not sure what should I do instead. I would appreciate if you could point some directions too, please.